Views/VMDisplayMetalViewController+Pointer.m

```diff
@@ -0,0 +1,116 @@
+//
+// Copyright © 2020 osy. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+#import "VMDisplayMetalViewController.h"
+#import "VMDisplayMetalViewController+Touch.h"
+#import "VMDisplayMetalViewController+Pointer.h"
+#import "VMCursor.h"
+#import "CSDisplayMetal.h"
+
+NS_AVAILABLE_IOS(13.4)
+@implementation VMDisplayMetalViewController (Pointer)
+
+- (void)addInteractionTo:(UIView*)view {
+    UIPointerInteraction *pointerInteraction = [[UIPointerInteraction alloc] initWithDelegate:self];
+    [view addInteraction:pointerInteraction];
+}
+
+// Add pointer interaction all interactive subviews except the keyboard
+-(void)initPointerInteraction {
+    [self addInteractionTo:self.mtkView];
+    [self addInteractionTo:self.powerExitButton];
+    [self addInteractionTo:self.pauseResumeButton];
+    [self addInteractionTo:self.restartButton];
+    [self addInteractionTo:self.zoomButton];
+    [self addInteractionTo:self.toggleKeyboardButton];
+    [self addInteractionTo:self.configureButton];
+    [self addInteractionTo:self.hideToolbarButton];
+    [self addInteractionTo:self.resumeBigButton];
+}
+
+#pragma mark - UIPointerInteractionDelegate
+- (UIPointerStyle *)pointerInteraction:(UIPointerInteraction *)interaction styleForRegion:(UIPointerRegion *)region {
+    // Hide cursor while hovering in VM view
+    if (interaction.view == self.mtkView) {
+        return [UIPointerStyle hiddenPointerStyle];
+    } else {
+        return [UIPointerStyle styleWithShape:[UIPointerShape shapeWithRoundedRect:interaction.view.bounds] constrainedAxes:UIAxisBoth];
+    }
+}
+
+static CGFloat CGPointToPixel(CGFloat point) {
+    return point * [UIScreen mainScreen].scale; // FIXME: multiple screens?
+}
+
+- (bool)isPointOnVMDisplay:(CGPoint)pos {
```

Comment:
`CGRectContainsPoint`?

Reply:
That didn't work because the rects that this function builds aren't sized properly. I didn't dive too deep into it.